### ISSUE TITLE
Feature Request: Multiple market-names, separated by a comma

### ISSUE THREAD

**Issue by rrarrarra** [Tucsky/SignificantTrades#113]
Sometimes, different exchanges use different "names" for markets (that are actually trading almost the same assets).

For example:

Bitmex: `"XBTUSD"` and FTX: `"BTCPERP"`.

or

FTX: `"UNI-PERP"` and Binance: `"UNIUSDT"`.

---

Question/Feature Request:
Is it possible to manually enter the market-names (separated by a comma) by myself, so that i do not have to open multiple windows, and therefore can have "equal" markets in only one window together? 

That would be really cool!!!

![Bildschirmfoto zu 2020-10-29 16-59-25](https://user-images.githubusercontent.com/5783257/97599309-20f63900-1a08-11eb-8c06-5ff33a1be916.png)

Thanks for help and best regards

**Comment by Tucsky** [comment]
Market names is complicated thing
Every exchange got their little way to name them, little "-" in there, or "/", one name it "perp" the other "perpetual", "swap".... (not even talking about the one that expires on a certain date, there is no hope for them)
The idea was to enter "btcusd" and it would match with the main ones automatically but in reality its just a fucking mess!
I think Im just gonna remove ALL the processing im doing on the names. Everybody will have to enter them the exact way, the way the exchanges want them to be named.

**Comment by Tucsky** [comment]
And about your request, just enter them separated by a "+" instead.

**Comment by rrarrarra** [comment]
Yes, trying to match all names automagically is quite impossible.

`UNI-PERP+UNIUSDT` works as expected.

Thanks
